enrolment-ui/src/modules/Login/components/GoogleLoginButton.js

```diff
@@ -0,0 +1,73 @@
+// adapted from: https://github.com/anthonyjgrove/react-google-login/blob/master/src/google-login.js
```

Comment:
In order to get the `access_token` required by our back-end, we have to use this library's "custom" button (the default, styled-out-of-the-box Login button does not return this token) which has no styles.

Rather than reinventing the wheel, I just adapted the button component the previous library was using. No one will suspect a thing ;)